packages/manager/modules/pci/src/projects/project/storages/databases/add/add.controller.js

```diff
@@ -59,10 +56,30 @@ export default class {
     this.apiGuideUrl =
       API_GUIDES[this.user.ovhSubsidiary] || API_GUIDES.DEFAULT;
     this.trackDatabases('configuration', 'page');
+
+    this.updateEngine(this.model.engine);
+
+    // If we find the partentElement, we can enable smooth scrolling. Otherwise, fallback to $anchor
+    const scrollParentElement = document.getElementsByClassName(
```

Comment:
Question: i don't have a personnel opinion, but is't a good practive using native `document` object instead `$document` ?

Reply:
I have changed it to `$document` for consistency.